env/export.go

```diff
@@ -87,40 +101,61 @@ func FromExport(body string) Environment {
 				continue
 			}
 
-			// Trim the `declare -x ` off the start of the line
-			line = strings.TrimPrefix(line, "declare -x ")
+			if strings.HasPrefix(line, "declare") {
+				command := strings.Split(line, " ")
```

Comment:
Could splitting on `" "` here and then doing `line = strings.Join(command[2:], " ")` later mangle env values with multiple contiguous spaces? Might we want to `strings.SplitN(line, " ", 3)` or similar so that we're only splitting the preamble/name, not the value?

Reply:
ooooooooh that's a really good callout, i never would've thought of that! will change it up.